test/browser/webgl_create_context_swapcontrol.c

```diff
@@ -23,5 +23,12 @@ int main() {
   EMSCRIPTEN_RESULT res = emscripten_webgl_make_context_current(context);
   assert(res == EMSCRIPTEN_RESULT_SUCCESS);
   assert(emscripten_webgl_get_current_context() == context);
+  emscripten_webgl_destroy_context(context);
+  // Test that creating a context for a canvas that's already been used once succeeds
+  context = emscripten_webgl_create_context("#canvas", &attrs);
+  assert(context > 0); // Must have received a valid context.
+  res = emscripten_webgl_make_context_current(context);
+  assert(res == EMSCRIPTEN_RESULT_SUCCESS);
+  assert(emscripten_webgl_get_current_context() == context);
```

Comment:
Should we `emscripten_webgl_destroy_context` here at the end?

Reply:
It wouldn't hurt, I'll add it.